test/test_dbcore.py

```diff
@@ -351,7 +351,7 @@ def pqp(self, part):
             part,
             {'year': dbcore.query.NumericQuery},
             {':': dbcore.query.RegexpQuery},
-        )
+        )[:-1]  # remove the negate flag
```

Comment:
Oh! I see it now. Thanks for explaining. This is fine; we can change the tests eventually, but this will do just fine for now.